### DISCUSSION TITLE
chrome上无法安装问题

### DISCUSSION THREAD

**Discussion by zhangya4548** [gildas-lormeau/SingleFile#1198] [Q&A]
chrome上无法安装,提示如下,但是检查了我的chrome没有安装过
![Xnip2023-02-28_11-47-49](https://user-images.githubusercontent.com/8876518/221760759-e9a437e2-ae8f-482c-b522-4579ca13f03e.png)

**Comment by gildas-lormeau** [comment]
Sorry, I don't speak Chinese.

**Comment by laoli987** [comment]
It is most likely a network problem
极有可能是网络问题

**Comment by zhangya4548** [comment]
> It is most likely a network problem 极有可能是网络问题

并不是网络问题,我安装别的插件正常

**Comment by xiaofeiTM233** [comment]
请检查谷歌的插件目录`.\User Data\Default\Extensions`下是否有`SingleFile(efnbkdcfmcmnhlkaijjjmhjjgladedno)`的文件目录，如果有，请删除
Please check the Google extensions folder`.\User Data\Default\Extensions`whether under a `SingleFile(efnbkdcfmcmnhlkaijjjmhjjgladedno)`folder, if is, please delete
临时解决方案：
Temporary solution:
你可以尝试打开开发人员模式
You can try turn on the "Developer Mode"
Then "Load the unzipped extension"(?我懒得切英文UI了好像是叫这个/I not switch to English UI but I think it's this?)
然后![image](https://user-images.githubusercontent.com/55832499/230113309-85eb0916-8ae4-47f1-8740-d2cead13f086.png)
这应该是谷歌的问题，将就着用吧
That should be Chrome's problem, so make do with it
